packages/shared/features/onboardingFeatures.ts

```diff
@@ -170,13 +170,13 @@ const onboardingFeatures = {
         devnet: {
             enabled: true,
             claimRewards: {
-                enabled: false,
```

Comment:
Should not be enabled until the routes are ready for beta testing

Reply:
I've re-disabled the flow, so it can't be accessed currently.